zonemaster/zonemaster-gui
# Update Finnish translation (FAQ mostly)

## Purpose

Try to fix the missing translation of the FAQ.

## Context

The translation system of the FAQ has changed. This import old translations into the new system.

## Changes

Finnish translations.

## How to test this PR

A quick check of the translated string would be welcome to verify that I haven't done any mistake while copy/pasting.

**Note:** It felt to me that some text differs from the original in content for some FAQ items, maybe it should be reviewed and updated accordingly to be more inline with the other languages. Also some strings are not translated.
